legendary_eureka.py

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env python3
 
+import json
+import random
+import time
+
 import requests
```

Comment:
Put all the `import` together then space and then `from x import y`  together

Reply:
PyCharm optimize imports function did that